mysql-test/suite/percona/t/component_binlog_utils_udf_gtid_tagged.test

```diff
@@ -0,0 +1,2 @@
+SET gtid_next="automatic:test_tag";
```

Comment:
Please put
```
--source include/have_udf.inc
--source include/have_grep.inc
```
in both `component_binlog_utils_udf_gtid_tagged.test` and `component_binlog_utils_udf_gtid_no_tag.test`
and remove it from the .inc file.

Reply:
Done